Source/DafnyCore/DafnyOptions.cs

```diff
@@ -195,7 +195,7 @@ public class DafnyOptions : Bpl.CommandLineOptions {
     }
 
     public DafnyOptions()
-      : base("Dafny", "Dafny program verifier", new DafnyConsolePrinter()) {
+      : base("dafny", "Dafny program verifier", new DafnyConsolePrinter()) {
```

Comment:
Why this change?

Reply:
It makes the output of `dafny audit --help` (and other command `--help` invocations) more correct. Previously, the help text made it appear that the name of the executable was `Dafny`.